source/turtles/turtle_hero.py

```diff
@@ -0,0 +1,137 @@
+import pygame, random
+
+GREEN = (20, 255, 140)
+GREY = (210, 210, 210)
+WHITE = (255, 255, 255)
+RED = (255, 0, 0)
+PURPLE = (255, 0, 255)
+BLACK = (0, 0, 0)
+WIDTH = 100
```

Comment:
Jeśli są nierozewralnie związane z zolwiem, to najlepiej przeniesc je bezposrednio do konstruktora zolwia
To dotyczy wszystkich tych constów poza kolorami